gobrew.go

```diff
@@ -675,6 +675,14 @@ func (gb *GoBrew) getGithubTags(repo string) (result []string) {
 	if repo == "golang/go" {
 		url = goBrewTagsApi
 	}
+
+	goTagsCacheDir := os.TempDir()
+	cachePath := goTagsCacheDir + strings.Replace(repo, "/", "_", -1) // replace / with _
+
+	// read from local cache in /tmp/go_tags
+	// skip error as it may have no cache
+	cachedData, _ := os.ReadFile(cachePath)
```

Comment:
You wrote `cachedData` here, but use it on line 718. Too much lines. :(

I think we need to move these lines below, as close as possible to where they are used.